dangerzone/gui/__init__.py

```diff
@@ -46,6 +54,26 @@ def monkeypatch_event(arg__1: QtCore.QEvent) -> bool:
 
         self.event = monkeypatch_event  # type: ignore [assignment]
 
+    def accumulate_file_open(self, filename: str) -> None:
```

Comment:
I'm not sure I understand why accumulating these events is necessary, if the application can handle adding extra documents at any time. Can you elaborate a bit?